scripts/contributing.js

```diff
@@ -36,9 +38,11 @@
                 labels: 'beginner-friendly'
             },
             success: function(data){
-                for(var i = 0; i < data.length; i++){
-                    $('.open-issues-section--beginner .issues').append('<li class="issue"><a href="' + data[i].html_url + '"><span class="issue__number">#' + data[i].number +'</span><span class="issue__title">' + data[i].title + '</span></a></li>');
-                }
+                var limitedData = _.take(data, ISSUES_LIMIT);
+
+                _.forEach(limitedData, function(issue) {
+                    $('.open-issues-section--beginner .issues').append('<li class="issue"><a href="' + issue.html_url + '"><span class="issue__number">#' + issue.number +'</span><span class="issue__title">' + issue.title + '</span></a></li>');
```

Comment:
I know this is similar to how it was before, but ideally we should remove the XSS vulnerabilities from this 😛 

We have jQuery available here, so this should **not** be building HTML elements like this:
```
<span class="issue__title">' + issue.title + '</span>
```